### PR TITLE
Feb 2021 Release

### PR DESCRIPTION
**Pull request by ansoncfit**

### CODE REVIEW

File: docs/guides/compute-freeform.mdx [hunk 1]
```diff
@@ -7,10 +7,98 @@ import {
...
+By default, Conveyal calculates accessibility indicators for every point in an analysis [grid](/analysis/methodology#spatial-resolution). 
 
-In certain cases, however, you may be interested in calculating accessibility indicators for certain non-grid "freeform" origins, or travel time indicators between a set of origins and destinations. There are a few prototype approaches for such calculations.
+You can override this default behavior to prepare CSV files with accessibility and travel-time results for a limited number of specified origins at exact coordinates. You first need to upload a CSV file and [process it as a freeform (non-grid) spatial dataset](/prepare-inputs/upload-spatial-data#freeform-non-grid-points). Next, in the custom request settings JSON of the analysis panel, add the fields described below to trigger the preparation of CSV result files. Then, create a [regional analysis]/analysis/regional) with Origin points set to a freeform spatial dataset. Once a regional analysis with freeform origins is complete, you will have options to download CSV files of the results. These features are still experimental; contact your support team for more information.
```
Comment (abyrd) [inline docs/guides/compute-freeform.mdx]: A single missing open paren in the link to regional:
```suggestion
You can override this default behavior to prepare CSV files with accessibility and travel-time results for a limited number of specified origins at exact coordinates. You first need to upload a CSV file and [process it as a freeform (non-grid) spatial dataset](/prepare-inputs/upload-spatial-data#freeform-non-grid-points). Next, in the custom request settings JSON of the analysis panel, add the fields described below to trigger the preparation of CSV result files. Then, create a [regional analysis](/analysis/regional) with Origin points set to a freeform spatial dataset. Once a regional analysis with freeform origins is complete, you will have options to download CSV files of the results. These features are still experimental; contact your support team for more information.
```

File: docs/glossary.mdx [hunk 2]
```diff
@@ -19,8 +19,8 @@ A modification is an alteration of some kind made to scheduled transit services.
 ## Network bundle
 A network bundle consists of an OpenStreetMap file and one or more [General Transit Feed Specification](#gtfs--gtfs-feed) (GTFS) feeds associated with your region. A region can have multiple bundles, and a bundle can include a single agency's GTFS or feeds from several adjacent or overlapping agencies.
 
-## Opportunity dataset
-An opportunity dataset is a spatial dataset with one or more numeric fields representing a count of opportunities (destinations) at particular locations in your [region](#region). See [uploading opportunities](/prepare-inputs/upload-opportunity-data#upload_opportunities).
+## Spatial dataset
+A spatial dataset has or more layers representing origin points or numeric counts of destination opportunities at particular locations in your [region](#region). See [uploading opportunities](/prepare-inputs/upload-spatial-data#upload_opportunities).
```
Comment (abyrd) [inline docs/glossary.mdx]:
```suggestion
A spatial dataset has one or more layers representing origin points or numeric counts of destination opportunities at particular locations in your [region](#region). See [uploading opportunities](/prepare-inputs/upload-spatial-data#upload_opportunities).
```

**Review by abyrd — COMMENTED** [review]
Looks good, I suggested a couple of corrections for small typos.

**Review by trevorgerhardt — APPROVED** [review]
🚀

**Pull request merged**